packages/profiling-node/src/integration.ts

```diff
@@ -305,6 +346,30 @@ class ContinuousProfiler {
     };
   }
 
+  /**
+   * Assigns thread_id and thread name context to a profiled event.
+   */
+  private _assignThreadIdContext(event: Event): any {
```

Comment:
honestly it might make sense to make this a default integration that works outside of profiling, so we always attach thread data to all sentry events (errors, transactions, spans).

This is because we are just relying on `worker_threads` for this info.

what do you think?

Reply:
Yeah, since the logic lives in core, I couldnt just add the import from node: and instead opted to add it here